_works/quovadis.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+category: work
+order: 7
+title: 'Quovadis Lisboa'
+subtitle: 'Mobile App & Website'
+client: 'Patriarcado de Lisboa'
+tags: ['mobile', 'website']
+image:
+ - url: '../../public/images/work/thumbs/moldura-quovadis.jpg'
+links:
+ - type: website
+   link: http://www.quovadislisboa.com/  
+ - type: iOS
+   link: https://itunes.apple.com/pt/app/quovadis-lisboa/id1222656489
+ - type: Android
+   link: https://play.google.com/store/apps/details?id=com.tplx.quovadis  
+sitemap: false
+---
+
+This is an app designed and built to help discover the various religious landmarks around the Lisbon area. 
```

Comment:
we can change the two lines to:

A digital guide for the analog religious tourism. Walk the walk, see the sights and enjoy churches, events and liturgies in the Lisbon metro area!